app/assets/javascripts/discourse/app/lib/logout.js

```diff
@@ -12,5 +12,5 @@ export default function logout({ redirect } = {}) {
     return;
   }
 
-  window.location.href = getURL("/");
```

Comment:
On login-required sites, visiting `/login` avoid directly triggering the login flow, which I think is what you're hoping to achieve here :+1:

However, on regular non-login-required sites, visiting `/login` will pop open the login modal. Generally that's not what people want after deleting an account. Instead, they'd want to visit the homepage of the site as an anonymous (logged-out) user.

I think the best strategy here is going to be to use a different URL for each case. We can check the login_required config via `ctx.siteSettings.login_required`

Reply:
thanks. a question for you. the instructions I'm following [here](https://github.com/discourse/discourse-oauth2-basic) say to use localhost:3000 vs. localhost:4200 for the google oAuth settings ... but it creates an awkward thing where after you login you get a landing page that says "use the following url to use Discourse" and then links to localhost:4200.

It sort of shakes my confidence that this is a "real" enough setup ... is there any way to get around this and have it simply redirect to localhost:4200? I'm getting an error when I just use localhost:4200 in the google oauth setup